### ISSUE TITLE
Setup.py has issues.

### ISSUE THREAD

**Issue by kshithijiyer** [google/TensorNetwork#13]

**Comment by kshithijiyer** [comment]
Patch Submitted: fd28c24

**Comment by kshithijiyer** [comment]
@sleichen could you please have a look at it and merge this patch?

**Comment by amilsted** [comment]
There's a typo in the patch that stops setup.py from executing:
```python
requirements = [r.strip() for requirement in
                open('requirements.txt').readlines()]
```

**Comment by kshithijiyer** [comment]
It's not exactly a spell mistake more of using two names where the variable name isn't properly address.

**Comment by kshithijiyer** [comment]
Fixed:  96e81ca
